biot_savart_integrators/generic/integrand.py

```diff
@@ -13,6 +14,16 @@
 def biot_savart_integrand(r, r_j, j, spatial_dim):
     R = r - r_j
     numerator = cross(j, R, spatial_dim)
-    denominator = (R**2).sum(dim=spatial_dim)**(3/2.)
+    denominator = (R**2).sum(dim=spatial_dim) ** (3 / 2.)
     integrand = _SI_FACTOR * numerator / denominator
-    return integrand
\ No newline at end of file
+    return integrand
+
+
+def biot_savart_potential_integrand(r: xr.DataArray, r_j: xr.DataArray,
+                                    j: xr.DataArray, spatial_dim: str):
+    R = r - r_j
+    numerator = j
+    denominator = (R ** 2).sum(dim=spatial_dim) ** (1 / 2.)
```

Comment:
Not sure the extra spaces around the operators make this more readable, for me it's the opposite. Is there some Numpy or Py convention?

Reply:
It is pycharm. I can return it...